README.md

```diff
@@ -57,56 +56,68 @@ All projects must meet these minimum requirements.
   * The GPL-3.0 license will prevent your project from one day being included in
     Backdrop core, so please only use this license when necessary.
 
+
 Backdrop Contributed Project Agreement
 --------------------------------------
 
 By joining the [Backdrop Contributed Project Group](https://github.com/backdrop-contrib)
 (Backdrop Contrib, for short) you agree to the following:
 
-1. You will not push changes to a repository for which you are not a current
-   maintainer (even though joining the Backdrop Contrib group will grant you
-   access to push to any project within).
+1. You will not push changes or merge PRs for a repository for which you are not
+   a current maintainer (unless you are a member of the Backdrop Bug Squad).
 
-1. You must agree to license your code contributions as GPLv2 or later.
+1. You must agree to license your code contributions as GPL-2.0 or GPL-3.0.
 
-1. Any project you create or maintain must include a copy of the [GPL-2.0
-   `LICENSE.txt` file](https://github.com/backdrop-ops/contrib/blob/master/examples/LICENSE.txt)
-   in the root of your repository. The GPLv2 license applies to all code that
-   directly interacts with parts of Backdrop licensed as GPLv2 or later. See the
-   [Backdrop License FAQ](https://backdropcms.org/license) for a more detailed
-   explanation.
+1. Any project you create or maintain must include a copy of the GPL-2.0 or
+   GPL-3.0 `LICENSE.txt` file in the root of your repository. The GPL license
```

Comment:
We should link to example files for both GPL 2 and 3 that people can copy

Reply:
We do, they are above.